lib/fluent/plugin/out_tdlog.rb

```diff
@@ -186,6 +186,8 @@ def format_stream(tag, es)
       out = ''
       off = out.bytesize
       es.each { |time, record|
+        next if record.nil? # Broken buffer sometimes nil record so such records should be skipped
```

Comment:
Yeah. I agree this patch is for ignoring records.
This is only for confirmation.